airbyte-cdk/python/build.gradle

```diff
@@ -13,8 +13,8 @@ task generateProtocolClassFilesWithoutLicense(type: Exec) {
     dependsOn ':tools:code-generator:airbyteDocker'
 }
 
-// todo: figure out dependencies
-task generateProtocolClassFiles(dependsOn: [generateProtocolClassFilesWithoutLicense, rootProject.spotlessPython])
-rootProject.spotlessPython.mustRunAfter(generateProtocolClassFilesWithoutLicense)
-rootProject.build.dependsOn generateProtocolClassFiles
-installReqs.dependsOn(generateProtocolClassFiles)
+task generateProtocolClassFiles(dependsOn: [generateProtocolClassFilesWithoutLicense])
+blackFormat.dependsOn generateProtocolClassFilesWithoutLicense
```

Comment:
i think this makes sense. 🤞  in other words any styling tasks have to happen _after_ any generation tasks.